tools/saqc/saqc.xml

```diff
@@ -399,451 +397,527 @@
                 <option value="ffill">ffill</option>
                 <option value="bfill">bfill</option>
               </param>
-              <param argument="flag" type="float" value="255.0" optional="true" label="Any, optional The flag value the function uses to mark observations" help="Defaults to the ``BAD`` value of the translation scheme"/>
-              <param argument="dfilter" type="float" value="-inf" optional="true" label="dfilter" help="Any, optional Defines which observations will be masked based on the already existing flags. Any data point with a flag equal or worse to this threshold will be passed as ``NaN`` to the function. Defaults to the ``DFILTER_ALL`` value of the translation scheme"/>
+              <param argument="flag" type="float" value="255.0" optional="true" label="flag" help="`Any`, optional The flag value the function uses to mark observations. Defaults to the ``BAD`` value of the translation scheme"/>
+              <param argument="dfilter" type="float" value="-inf" optional="true" label="dfilter" help="`Any`, optional Defines which observations will be masked based on the already existing flags. Any data point with a flag equal or worse to this threshold will be passed as ``NaN`` to the function. Defaults to the ``DFILTER_ALL`` value of the translation scheme"/>
             </when>
             <when value="transferFlags">
-              <param argument="field" type="text" value="" label="Field" help="The name of the variable to process.">
-                <validator type="empty_field"/>
-              </param>
-              <param argument="target" type="text" optional="true" label="Target" help="The name of the variable to process."/>
-              <param argument="squeeze" type="boolean" label="squeeze" help="Squeeze the history into a single column if ``True``, function specific flag information is lost" checked="false" truevalue="squeeze" falsevalue=""/>
-              <param argument="overwrite" type="boolean" label="Overwrite existing flags if ``True``" help="Overwrite existing flags if ``True``" checked="false" truevalue="overwrite" falsevalue=""/>
+              <param argument="field" type="data_column" label="field" help="Variable to process" data_ref="data" multiple="false"/>
+              <param argument="target" type="data_column" optional="true" label="target" help="`SaQCFields` | `newSaQCFields` , optional Variable name to which the results are written. `target` will be created if it does not exist. Defaults to `field`" data_ref="data" multiple="false"/>
+              <param argument="squeeze" type="boolean" label="squeeze" help="bool If True, compress the history into a single column, losing function-specific flag information" checked="false" truevalue="squeeze" falsevalue=""/>
```

Comment:
Are `truevalue` and `falsevalue` needed for booleans. In the json we have `True`/`False` anyway, or?